### DISCUSSION TITLE
请问如何填写 X-MOL 期刊源的 type 参数？

### DISCUSSION THREAD

**Discussion by fountainer** [DIYgod/RSSHub#8369] [Q&A] [answered]
比如想要订阅 https://www.x-mol.com/paper/journal/5115 这个期刊，感觉  magzine 参数是 5115, 但 type 参数是多少。给的例子中  https://rsshub.app/x-mol/paper/0/9 这个type 参数 0, 不知道来自哪里？

**Comment by auto-bot-ty** [comment, marked as answer]
https://rss.shab.fun/x-mol/paper/0/5115
`type` 参数失效，  `https://www.x-mol.com/paper/0/9` 重定向至 `https://www.x-mol.com/paper/journal/9 `

**Reply by fountainer** [reply]
谢谢！
